cli/integrity/integrity.go

```diff
@@ -0,0 +1,25 @@
+package integrity
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// IntegritySigner is a noop integrity check preparation.
+type IntegritySigner struct{}
```

Comment:
It's suitable to make an interface instead of the concrete implementation.

Reply:
I had made an interface at some point, but it feels redundant to me. Having it as a part of `integrity` module is inconvenient, since the whole module gets replaced via `go.mod` file, we will have to duplicate it both here and in actual implementation. Moving it to a separate package, like `integrity_interface` also feels unnecessary -- it seems that there is only one implementation of integrity module per project. Even if we resort to having various hashing and signature algorithms, I would rather prefer to parametrize over them rather than provide several implementations of the module.

Reply:
Just an interface in the `integrity` module will be enough. This will help make the logic more agile in the future without additional steps:

Create different specific implementations from the `NewIntegritySigner` method depending on something in the tt-ee part.

Reply:
I've created an appropriate interface.